src/interface/storage.thrift

```diff
@@ -163,6 +163,12 @@ struct Edge {
     2: binary props,
 }
 
+struct IndexItem {
+     1: required common.IndexID          index_id,
+     2: required i32                     schema,
```

Comment:
Why "schema" type is i32? 

It seems the name should be tagIdOrEdgeType.

Reply:
A better name will be helpful when reading your code.  
So we'd better give a more meaningful name here.  
How about "tagOrEdge", short and meaningful.

Reply:
> A better name will be helpful when reading your code.
> So we'd better give a more meaningful name here.
> How about "tagOrEdge", short and meaningful.

I agree with `tagOrEdge`